packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -61,7 +60,7 @@ export const Tab = React.forwardRef<HTMLAnchorElement, TabProps>(
 
 Tab.displayName = 'Tab';
 
-const getTabStyles = stylesFactory((theme: GrafanaTheme2) => {
+const getStyles = (theme: GrafanaTheme2) => {
```

Comment:
There's something I found confusing in the styles here: we apply some styling rules to `a` that are nested in `.link` on lines 96 and 111 (85 and 109 in `main`) and I'm not sure that's valid. Unless someone passes an anchor element to the `suffix` prop, but an an anchor element [shouldn't be a descendant of another anchor element](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element).